tests/Slug/ValidCharactersTest.php

```diff
@@ -50,23 +54,43 @@ function (SlugValidCharactersEvent $event) use (&$path): bool {
             )
         ;
 
-        $validCharacters = new ValidCharacters($eventDispatcher);
+        $translator = $this->createMock(TranslatorInterface::class);
 
-        $GLOBALS['TL_LANG']['MSC']['validCharacters'] = [
-            'unicodeLowercase' => 'unicodeLowercase',
-            'unicode' => 'unicode',
-            'asciiLowercase' => 'asciiLowercase',
-            'ascii' => 'ascii',
-        ];
+        $translator
+            ->expects($this->at(0))
+            ->method('trans')
+            ->with('MSC.validCharacters.unicodeLowercase')
+            ->willReturn('Unicode numbers and lowercase letters')
+        ;
 
+        $translator
+            ->expects($this->at(1))
```

Comment:
Is it necessary to test how the tranlator is used?

Reply:
Probably not. 😄

Reply:
Although the test assures that the class iterates over all four default options.

Reply:
But the order should not be relevant. I think we should remove the translator tests here.